build/DotNet.fs

```diff
@@ -27,7 +27,9 @@ module DotNet =
         if Environment.isUnix then
             Environment.environVar "HOME" @@ ".dotnet"
         else
-            Environment.environVar "LocalAppData" @@ "Microsoft" @@ "dotnet"
+            Environment.environVar "LocalAppData"
+            @@ "Microsoft"
+            @@ "dotnet"
```

Comment:
Did any logic in this file change, or is it only formatting changes?

Reply:
Only formatting.

Only this added
https://github.com/fsprojects/FSharp.Data.GraphQL/blob/76af489fe2ef701754827d34e38a02bab56021e2/build/NuGet.fs#L576-L578